src/lib/utils.ts

```diff
@@ -27,26 +22,16 @@ export const getCurrentDate = (): string => {
   return `${year}-${month}-${day} ${hours}:${minutes}:${seconds}.${milliseconds}${nanoseconds} UTC`
 }
 
-const byteConverter = new ByteConverter()
-
 /**
  * This function is used to convert string formatted bytes to bytes
  *
  * @param inputDatacap
  * @returns number
  */
 export function anyToBytes(inputDatacap: string): number {
-  const formatDc = inputDatacap
-    .replace(/[t]/g, 'T')
-    .replace(/[b]/g, 'B')
-    .replace(/[p]/g, 'P')
-    .replace(/[I]/g, 'i')
-    .replace(/\s*/g, '')
-  const ext = formatDc.replace(/[0-9.]/g, '')
-  const datacap = formatDc.replace(/[^0-9.]/g, '')
   try {
-    const bytes = byteConverter.convert(parseFloat(datacap), ext, 'B')
-    return bytes
+    const parsedBytes = bytes.parse(inputDatacap)
```

Comment:
When can parsedBytes be empty?
I think empty parsedBytes should be treated as an error - so even if we fallback to 0 we should log that it happened.